src/esss_fix_format/cli.py

```diff
@@ -374,9 +398,14 @@ def _main(files_or_directories, check, stdin, commit, pydevf_format_func, *, ver
     files = sorted(Path(x) for x in files)
     errors = []
 
-    black_config = find_black_config(files)
+    pyproject_toml = find_pyproject_toml(files)
+
+    if pyproject_toml:
+        global EXCLUDE_PATTERNS
```

Comment:
Please get rid of the global, and pass the list of patterns explicitly

Reply:
I can't see harm of globals in this context (a single module shell script). 

But let's stick with the good practice of avoiding mutable globals. I removed the global variable and changed `should_format` to receive both the list of included and excluded patterns.